examples/templates/aws-linux/main.tf

```diff
@@ -291,7 +279,7 @@ resource "aws_instance" "dev" {
   availability_zone = "${data.coder_parameter.region.value}a"
   instance_type     = data.coder_parameter.instance_type.value
 
-  user_data = data.coder_workspace.me.transition == "start" ? local.user_data_start : local.user_data_end
+  user_data = data.coder_workspace.me.transition == "start" ? local.user_data_start : null
```

Comment:
I don't think we need to update the user data based on the transition state anymore. It can be hardcoded to local.user_data